### DISCUSSION TITLE
wazuh docker install remote syslog debug

### DISCUSSION THREAD

**Discussion by capitangiaco** [wazuh/wazuh#17593] [Q&A]
hi 

I am trying to collect and analyze logs from firewalls to wazuh (docker single-node install)

~/wazuh-docker/single-node # docker ps
CONTAINER ID   IMAGE                         COMMAND                  CREATED          STATUS          PORTS                                                                                                                                                           NAMES
5a4f7a5723a5   wazuh/wazuh-dashboard:4.4.3   "/entrypoint.sh"         37 minutes ago   Up 36 minutes   443/tcp, 0.0.0.0:443->5601/tcp, :::443->5601/tcp                                                                                                                single-node_wazuh.dashboard_1
cec50a96a29b   wazuh/wazuh-manager:4.4.3     "/init"                  37 minutes ago   Up 36 minutes   0.0.0.0:1514-1515->1514-1515/tcp, :::1514-1515->1514-1515/tcp, 0.0.0.0:514->514/udp, :::514->514/udp, 0.0.0.0:55000->55000/tcp, :::55000->55000/tcp, 1516/tcp   single-node_wazuh.manager_1
d604936e274b   wazuh/wazuh-indexer:4.4.3     "/entrypoint.sh open…"   37 minutes ago   Up 36 minutes   0.0.0.0:9200->9200/tcp, :::9200->9200/tcp                                                                                                                       single-node_wazuh.indexer_1

ossec.conf
....
 ```
 <remote>
    <connection>secure</connection>
    <port>1514</port>
    <protocol>tcp</protocol>
    <queue_size>131072</queue_size>
  </remote>

  <remote>
    <connection>syslog</connection>
    <port>514</port>
    <protocol>udp</protocol>
    <allowed-ips>0.0.0.0/0</allowed-ips>
  </remote>
.
```.....


with 'tcpdump -i any port 514' I can see devices communicate with the docker server and the wazuh manager container:

12:05:48.371402 eth0  In  IP 192.168.231.254.57615 > wazuh.*.*.it.514: SYSLOG user.info, length: 104
12:05:48.371477 br-73a73a9fa4bc Out IP 172.19.0.1.49434 > 172.19.0.3.514: SYSLOG user.info, length: 104
12:05:48.371481 veth35e5dbc Out IP 172.19.0.1.49434 > 172.19.0.3.514: SYSLOG user.info, length: 104
12:05:49.837783 eth0  In  IP Mjolnir.*.*.it.514 > wazuh.*.*.it.514: SYSLOG local0.info, length: 159
12:05:49.837798 br-73a73a9fa4bc Out IP Mjolnir.*.*.it.514 > 172.19.0.3.514: SYSLOG local0.info, length: 159

In the Discovery tab I find only wazuh-agents logs, how can debug it ?

thanks

**Comment by mdiego92** [comment]
Hello @capitangiaco,

If you're receiving data from the firewall, then we need to verify that Wazuh is reading the information properly. For this, please enable the [logall_json](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#logall-json) option on your manager's `/var/ossec/etc/ossec.conf`:

```
<global>
  <jsonout_output>yes</jsonout_output>
  <alerts_log>yes</alerts_log>
  <logall>no</logall>
  <logall_json>yes</logall_json>
...
```

Restart the manager for the changes to take effect:
`systemctl restart wazuh-manager`

Then you can check if you are getting the devices logs in the /var/ossec/logs/archives/archives.json file:
`tail -f /var/ossec/logs/archives/archives.json | grep -i "agentless"`

If you see any results from it, could you please send me a few examples of the full_log field of the events from your device?
Finally, please disable the `logall_json` setting to avoid any issues due to excessive disk space usage.

Thank you very much.
